src/components/Transaction.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
+  const transaction = props.transaction
   return (
-    <tr>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
+    <tr key={transaction.id}>
```

Comment:
whoops, yeah, this is leftover from before i abstracted the transactions away from the TransactionsList.js #sloppy